server.js

```diff
@@ -111,6 +112,20 @@ const startServer = (html) => {
     if (env === 'local' || env === 'opplaering') {
         console.log('Setter opp lokale mock-endepunkter');
         require('./Mock/mockEndepunkter').mockForLokal(server);
+    } else {
+        server.use('/api', ((req, res, next) => {
+            console.log("Proxy request to backend for " + req.path);
+            next();
```

Comment:
hele denne blokken kan fjernes igjen, den var hovedsaklig for å sjekke om vi i det hele tatt traff handleren.